sdk/eventhub/eventhubs-checkpointstore-tables/sample.env

```diff
@@ -0,0 +1,18 @@
+# Used in most samples
+EVENTHUB_CONNECTION_STRING=<connection string WITHOUT an EntityPath>
+EVENTHUB_NAME=<name of a single eventhub>
+EVENTHUB_FQDN=<your-eventhubs-namespace>.servicebus.windows.net
+CONSUMER_GROUP_NAME=<name of a consumer group>
+STORAGE_CONNECTION_STRING=<connection string for a storage account for the checkpoint store>
+
+# Used in the useWithIotHub.ts sample
+IOTHUB_EH_COMPATIBLE_CONNECTION_STRING=<connection string with EntityPath>
+
+# Used to authenticate using Azure AD as a service principal for role-based authentication
+# in the `usingAadAuth` sample.
+#
+# See the documentation for `EnvironmentCredential` at the following link:
+# https://docs.microsoft.com/javascript/api/@azure/identity/environmentcredential
+AZURE_CLIENT_ID=
+AZURE_TENANT_ID=
+AZURE_CLIENT_SECRET=
```

Comment:
For now we don't have any samples. Let's remove this file and we can add it back in once we have samples/tests that hit the real service.